synapseclient/client.py

```diff
@@ -1543,6 +1691,11 @@ def store(
                             You will be contacted with regards to the specific data being restricted and the
                             requirements of access.
             set_annotations: If True, set the annotations on the entity. If False, do not set the annotations.
+            async_file_handle_upload: Temporary feature flag that will be removed at an
```

Comment:
Nit: not to complicate things, but is it worth doing a `_async...` parameter?

Reply:
Not for this. I intend that this is removed with https://sagebionetworks.jira.com/browse/SYNPY-1356 - Which is not that far away.